packages/ui-library/src/components/actions/buttons/icon-button/index.ts

```diff
@@ -23,25 +23,25 @@ export class BlrIconButton extends LitElement {
   @property() icon?: SizelessIconType;
   @property() loading?: boolean;
   @property() disabled!: boolean;
-  @property() buttonId?: string;
+  @property() iconButtonId?: string;
   @property() variant: ActionVariantType = 'primary';
-  @property() size?: ActionSizesType = 'md';
+  @property() sizeVariant?: ActionSizesType = 'md';
   @property() loadingStatus!: string;
 
   @property() theme: ThemeType = 'Light';
 
   // these are not triggered directly but allows us to map it internally and bve typesafe
-  @property() blrFocus?: () => void;
-  @property() blrBlur?: () => void;
-  @property() blrClick?: () => void;
+  @property() onFocus?: () => void;
```

Comment:
we have to talk about this part :)

Yes, the ticket may suggest to do that, but when implementing the example app, we found out that it is highly recommended/needed to have a different event name, then standard html.

Because the inner html button will fire onBlur etc as well, it will bubble up and then you will end up having event listeners called twice :(